pkg/alibabacloud/eni/node_test.go

```diff
@@ -122,33 +130,35 @@ func (e *ENISuite) TestCandidateAndEmtpyInterfaces(c *check.C) {
 	n := &Node{}
 	n.k8sObj = cn
 	// Primary ENI excluded, max allocatable = 3 ( 1 (ENI) * 3 (IPv4/ENI) )
-	c.Assert(n.GetMaximumAllocatableIPv4(), check.Equals, 3)
+	require.Equal(t, n.GetMaximumAllocatableIPv4(), 3)
 
 	// Wait for IPs to become available
-	c.Assert(testutils.WaitUntil(func() bool { return reachedAddressesNeeded(mngr, "node3", 0) }, 5*time.Second), check.IsNil)
+	require.NoError(t, testutils.WaitUntil(func() bool { return reachedAddressesNeeded(mngr, "node3", 0) }, 5*time.Second))
```

Comment:
nit: you can use `require.Eventually` here.

```suggestion
	require.Eventually(t, func() bool { return reachedAddressesNeeded(mngr, "node3", 0) }, 5*time.Second, 1*time.Second)
```

Reply:
ah nice :+1: